packages/kotti-ui/source/kotti-comment/KtComment.vue

```diff
@@ -105,16 +108,19 @@ export default defineComponent<KottiComment.PropsInternal>({
 		const userBeingRepliedTo = ref<UserData | null>(null)
 		const translations = useTranslationNamespace('KtComment')
 
-		const handleDelete = (commentId: number | string, isInline?: boolean) => {
+		const handleDelete = (
+			commentId: number | string | null,
```

Comment:
I don't think null makes sense here. How can you delete a comment without an ID?